CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
+- Added support for vertical X axis to HeatMapSeries (#535)
```

Comment:
It is not, though the example for issue #535 does demonstrate it neatly at this time.